R/create_extendr_package.R

```diff
@@ -0,0 +1,80 @@
+
+#' Create a project for R package development with Rust
+#'
+#' @description
+#' This function creates an R project directory for package development
+#' with Rust extensions.
+#'
+#' @param path a path to new directory
+#' @param ... arguments passed on to `usethis::create_package()` and
+#' `rextendr::use_extendr()`
+#'
+#' @return Path to the newly created project or package, invisibly.
+#' @keywords internal
+#'
+#' @examples
+create_extendr_package <- function(path, ...) {
```

Comment:
Should we mark it as `@noRd`? @CGMossa what's your take?

Reply:
If it's internal, will roxygen still generate a documents page unless we explicitly use `@noRd`?

Reply:
Just check it, did you run `devtools::document()`?

Reply:
I did not! Is that something you want all contributors to do for each pull request? Should I also do `devtools::check()`?

Reply:
Document - yes, since we do not commit changes on CI. Check - up to you, but this is done on CI and it is ✅